docs/components/footer.vue

```diff
@@ -43,6 +43,49 @@
 
       <hr>
 
+      <template v-if="isZeitNow">
```

Comment:
@tmorehouse We will receive the proper assets to use next week. Maybe we should wait with that until then.

Reply:
I've temporarily set the `v-if` to `false` until we get the new logo